### PR TITLE
New Swimm Units

### PR DESCRIPTION
**Pull request by shreyamalviya**
Adds 3 new Swimm units for a new playlist (_Adding a PBA in 5 Simple Steps, for Open Sauced_):
- _Define what your new PBA does_
- _Implement a new PBA — `ScheduleJobs`_
- _Add details about your new PBA_

### CODE REVIEW

File: .swm/JFXftJml8DpmuCPBA9rL.swm
Comment (VakarisZ) [inline .swm/JFXftJml8DpmuCPBA9rL.swm:30]: Newline?

File: .swm/VW4rf3AxRslfT7lwaug7.swm
Comment (VakarisZ) [inline .swm/VW4rf3AxRslfT7lwaug7.swm:3]: don't we already have this PBA? Maybe we should instead start with a more trivial task, like a PBA that adds a text tile to temp dir?

**Review by VakarisZ — CHANGES_REQUESTED** [review]
Can't approve until questions are answered

File: .swm/JFXftJml8DpmuCPBA9rL.swm
Comment (shreyamalviya) [reply .swm/JFXftJml8DpmuCPBA9rL.swm:30]: Ah, Swimm committed these files. I reckon we should let these be just for consistency? Otherwise we'll have to edit the Swimm commit and add a newline every time.

File: .swm/VW4rf3AxRslfT7lwaug7.swm
Comment (shreyamalviya) [reply .swm/VW4rf3AxRslfT7lwaug7.swm:3]: Maybe we can add more playlists/units for trivial tasks; this one was used in the OpenSauced presentation so we should probably keep this separate.

Comment (shreyamalviya) [reply .swm/VW4rf3AxRslfT7lwaug7.swm:3]: Actually, this doesn't require the swimmer to add the actual job scheduling commands. They're only required to fetch the commands from an already existing function, `get_commands_to_schedule_jobs`, which is mentioned in the unit's description. So it's not really hard.

**Review by VakarisZ — APPROVED** [review]
Approved, you can merge if that's all you wanted to add in this PR

**Review by freemason88 — APPROVED** [review]

**Pull request merged**
